v3/src/components/slider/slider-model.ts

```diff
@@ -22,7 +22,7 @@ export const SliderModel = TileContentModel
     animationMode: types.optional(types.enumeration([...AnimationModes]), kDefaultAnimationMode),
     // clients should use animationRate view defined below
     _animationRate: types.maybe(types.number),  // frames per second
-    axis: types.optional(NumericAxisModel, { place: 'bottom', min: 0, max: 12 })
+    axis: types.optional(NumericAxisModel, { place: 'bottom', min: -0.9, max: 11.78 })
```

Comment:
Out of curiosity, are these the actual v2 defaults?

Reply:
No, Bill has specified them, and has since corrected the spec.